DataScience/ImportantFeatures.py

```diff
@@ -113,7 +112,7 @@ def get_important_features(log_file, ml_args, warmstart_model=None, min_num_feat
     while True:
         readModel_fp = readModel_fp_base.format(index)
```

Comment:
I would use directly `log_file+'.readModel.{0}.txt'` here for readibility removing the variable `readModel_fp_base`